homopolymer_fit/README_homopolymer_fit.md

```diff
@@ -1,11 +1,7 @@
+# Homopolymer fit
+
 This is a series of scripts for Ising analysis of homopolymeric NRC-type repeats.  There are three separate .py files
 for data conversion, generation of fitting equations, and fitting and downstream processing (plotting, boostrapping,
 statistical analysis, and parameter correlation).
 
-In addition, there is a single jupyter .ipynb notebook that combines all of these functions.  
-
-Finally, there is a set of data in its own folder for fitting.  When fitting, the directory that houses these data should be 
-used as the path in each script (usually I put versions of the script(s) in the same folder, so they have I can review
-any post-fit settings at a later time).
-
-These run in python 2.7
```

Comment:
@barricklab-at-jhu you see I've already removed these python 2 comments from those "internal" readme's